extensions/artifacts/org.wso2.developerstudio.codenvy.ext.appfactory/src/main/java/org/wso2/developerstudio/codenvy/ext/appfactory/client/resources/AppFactoryExtensionResources.java

```diff
@@ -24,4 +24,9 @@
 public interface AppFactoryExtensionResources extends ClientBundle {
```

Comment:
We have added a comment to this interface and you can look into that from the latest code at 
https://github.com/wso2/cloud-dev-studio/blob/master/extensions/artifacts/org.wso2.developerstudio.codenvy.ext.appfactory/src/main/java/org/wso2/developerstudio/codenvy/ext/appfactory/client/resources/AppFactoryExtensionResources.java